pkgs/applications/networking/listadmin/default.nix

```
@@ -0,0 +1,46 @@
+{ lib, stdenvNoCC, fetchurl, makeWrapper, perl, installShellFiles }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "listadmin";
+  version = "2.73";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/project/listadmin/${version}/listadmin-${version}.tar.gz";
+    sha256 = "00333d65ygdbm1hqr4yp2j8vh1cgh3hyfm7iy9y1alf0p0f6aqac";
+  };
+
+  buildInputs = [ perl ];
+  nativeBuildInputs = [ makeWrapper installShellFiles ];
+
+  # There is a Makefile, but we don’t need it, and it prints errors
+  dontBuild = true;
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share/man/man1
+    install -m 755 listadmin.pl $out/bin/listadmin
+    installManPage listadmin.1
+
+    wrapProgram $out/bin/listadmin \
+      --prefix PERL5LIB : "${with perl.pkgs; makePerlPath [
+        HTMLParser LWP LWPUserAgent MIMEBase64 HTMLTagset
+        HTTPMessage URI HTTPDate TryTiny TextReform
+        IOSocketSSL NetSSLeay NetINET6Glue EncodeLocale
+        LWPProtocolhttps NetHTTP
+        ]}"
+  '';
+
```

Since `listadmin` doesn't include a test suite, we can include an install test that checks that the script runs. 

```suggestion

  doInstallCheck = true;
  installCheckPhase = ''
    $out/bin/listadmin --help 2> /dev/null  
  '';

```

---

Will do!

---

Added (and tested that it fails if I miss a perl dependency)